### DISCUSSION TITLE
MaskPhish not link not working

### DISCUSSION THREAD

**Discussion by lharbor0** [jaykali/maskphish#79] [Q&A]
I installed just how it says and bash the maskphish.sh and input the ngrok url and type all the custom URL stuff in, but when it generates the custom URL which becomes (https://snapchat.com-login-verification-support-mobile-email@) the site doesn't work/ doesn't connect to my ngrok original url?? I am running Kali Linux Debian (64bit) please help!

**Comment by jaykali** [comment]
We use [https://is.gd](https://is.gd) URL shortner services to give a proper masked look. But we think either the service is down or your Internet service provider is blocking [https://is.gd](https://is.gd) services. You can try opening [https://is.gd](https://is.gd) from your browser. It will not open. Use A VPN or proxy etc to use the services, if not working then the is.gd services is down. In that case you might need to wait for some hour maybe a day.

Thanks.

**Reply by humdrumsanagujar** [reply]
Please convert is.gd into da.gd i Open VPN for loction changes  ip adresse please solve this problem i wait for your new version
